src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace TEC\Tickets\Commerce;
+
+class Payments_Tab {
+
+	/**
+	 * The option key to check TicketsCommerce status.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_activated;
```

Comment:
We shouldn't store the key separately here. Use the `\TEC\Tickets\Settings::$tickets_commerce_enabled` directly.